prescyent/dataset/motion/teleop_icub/dataset.py

```diff
@@ -0,0 +1,117 @@
+"""Class and methods for the TeleopIcub Dataset
+https://zenodo.org/record/5913573#.Y75xK_7MIaw
+"""
+from pathlib import Path
+from typing import Callable, List, Union, Dict
+
+import numpy as np
+import torch
+
+from prescyent.dataset.motion.episodes import Episode
+from prescyent.utils.logger import logger, DATASET
+from prescyent.dataset.motion.dataset import MotionDataset, Episodes
+from prescyent.utils.dataset_manipulation import split_array_with_ratios
+from prescyent.dataset.motion.teleop_icub.config import TeleopIcubDatasetConfig
+
+
+TELEOP_DIMENSIONS_NAMES = [
+    "waist_z", "right_hand_x", "right_hand_y", "right_hand_z",
+    "left_hand_x", "left_hand_y", "left_hand_z"
+]
+
+
+class TeleopIcubDataset(MotionDataset):
```

Comment:
I am not sure I like this naming convention: why not Dataset? Since we are in `motion.teleop_icub`, I would prefer to write:
`import motion.teleop_icub.Dataset as icub_dataset`

At any rate, this is quite redundant naming.